RealmBrowser/Classes/RLMRealmBrowserWindowController.m

```diff
@@ -101,6 +101,20 @@ - (IBAction)saveObjcModels:(id)sender
     [RLMModelExporter saveModelsForSchemas:objectSchemas inLanguage:RLMModelExporterLanguageObjectiveC];
 }
 
+- (IBAction)saveCopy:(id)sender
+{
+    NSString *fileName = [self.modelDocument.presentedRealm.realm.path lastPathComponent];
+    NSSavePanel *panel = [NSSavePanel savePanel];
+    [panel setNameFieldStringValue:fileName];
+    [panel beginSheetModalForWindow:self.window completionHandler:^(NSInteger result){
+        if (result == NSFileHandlingPanelOKButton)
+        {
+            NSURL *fileURL = [panel URL];
+            [self.modelDocument.presentedRealm.realm writeCopyToPath:fileURL.path error:nil];
```

Comment:
Fair enough! I've added error handling now.

For now, I just exposed `[RLMRealm compact]` and had that run on the new Realm file. Are there any downsides to that?